plugins/rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -840,6 +840,48 @@ types:
       title: Remediated Vulnerabilities
       type: '[]string'
       required: false
+  exploits:
+    description:
+      title: Description
+      description: A verbose description of the exploit
+      type: string
+      required: true
+    id:
+      title: ID
+      description: The identifier of the exploit
+      type: string
+      required: true
+    name:
+      title: Name
+      description: The name of the exploit
+      type: string
+      required: true
+    rank:
+      title: Rank
+      description: The exploit rank
+      type: string
+      enum:
```

Comment:
Because this is used only in outputs can we remove the enum to prevent this breaking in future updates to InsightVM?

Reply:
yes, good idea